File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/structure/onetomany/OneToManyModelNonModel.java
package com.raizlabs.android.dbflow.test.structure.onetomany;

import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.test.TestDatabase;

/**
 * Description:
 *
 * @author Andrew Grosner (fuzz)
 */
@Table(database = TestDatabase.class)
public class OneToManyModelNonModel {

    @PrimaryKey
    int id;

    @PrimaryKey
    String name;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/structure/onetomany/OneToManyModelNonModel.kt
package com.raizlabs.android.dbflow.test.structure.onetomany

import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.test.TestDatabase

/**
 * Description:

 * @author Andrew Grosner (fuzz)
 */
@Table(database = TestDatabase::class)
class OneToManyModelNonModel {

    @PrimaryKey
    var id: Int = 0

    @PrimaryKey
    var name: String
}
